libraries/app/application.cpp

```diff
@@ -1027,7 +1028,7 @@ void application::initialize(const fc::path& data_dir, const boost::program_opti
       }
       fc::json::save_to_file(genesis_state, genesis_out);
 
-      std::exit(EXIT_SUCCESS);
+      return false;
```

Comment:
`EXIT_SUCCESS` is lost. Perhaps better to return an integer rather than a bool.

Reply:
Great point. Fixed.